css-two-column-layout/styles.css

```diff
@@ -0,0 +1,129 @@
+.container {
+  max-width: 1024px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  justify-content: right;
+}
+
+.column-full {
+  flex-basis: 100%;
+  justify-content: center;
+  text-align: center;
+}
+
+h1 {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 0.8rem;
+}
+
+p {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 0.75rem;
+  line-height: 1.5rem;
+}
+
+.header > h1 {
+  font-size: 250%;
+  letter-spacing: 5px;
+}
+
+.header > p {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 250;
+  line-height: 25px;
+  font-size: 0.85rem;
+  margin-top: -20px;
+  color: rgb(105 105 105);
+}
+
+.column-half {
+  flex-basis: 50%;
+}
+
+.column-third {
+  flex-basis: calc(100%/3);
+}
+
+.column-two-thirds {
+  flex-basis: calc(200%/3);
+}
+
+#top-row {
+  margin-right: 5rem;
+}
+
+#flexbox-tl {
+  margin-top: 2rem;
+}
+
+img {
+  height: 125px;
+  width: 125px;
+  margin-bottom: 1rem;
+  margin-left: 2rem;
+}
+
+#image-tl {
+  margin-left: 2rem;
+  margin-bottom: 4rem;
+}
+
+#flexbox-tr {
+  margin-top: 1.5rem;
+  margin-right: 0;
+}
+
+#image-tr {
+  margin-left: 4rem;
+  margin-bottom: 4rem;
+}
+
+#bottom-row {
+  margin-right: 5rem;
+}
+
+#flexbox-bl {
+  margin-top: 2rem;
+}
+
+#image-bl {
+  margin-left: 2rem;
+  margin-bottom: 4rem;
+}
+
+#flexbox-br {
+  margin-top: 1.5rem;
+  margin-right: 0;
+}
+
+#image-br {
+  margin-left: 4rem;
+  margin-bottom: 4rem;
+}
+
+#final-row {
+  margin-top: 5px;
+  margin-bottom: 3rem;
+}
+
+input {
+  padding: 10px 3rem;
+  background-color: white;
+  border: solid red;
+}
+
+[value] {
```

Comment:
This selector is kind of confusing, I would just add a class to the submit button that you can apply this styling to.